apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -40,18 +40,18 @@ function original_request_url() {
 }
 
 /**
- * Returns a redirect URL for post-login flow in WordPress.com
+ * Returns an unencoded redirect URL for post-login flow in WordPress.com.
  *
  * @return string The redirect URL
  */
 function get_wpcom_redirect_to() {
 	// Redirect to the current URL.
 	// If, for any reason, the superglobals aren't available, set a default redirect.
 	if ( empty( $_SERVER['HTTP_HOST'] ) || empty( $_SERVER['REQUEST_URI'] ) ) {
-		return get_marketing_home_url();
+		return get_onboarding_url();
 	}
 
-	return rawurlencode( 'https://' . sanitize_text_field( wp_unslash( $_SERVER['HTTP_HOST'] ) ) . sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) );
+	return 'https://' . sanitize_text_field( wp_unslash( $_SERVER['HTTP_HOST'] ) ) . sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) );
```

Comment:
The encoding/sanitizing occurs in the WPCOM filter now. This is to ensure that we don't have to remember to encode the value wherever we use the filter.